scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/job/Job.java

```diff
@@ -86,6 +88,11 @@ public abstract class Job extends CommonAttribute {
      */
     protected String projectName = "";
 
+    /**
+     * Tags of this job
+     */
+    protected Set<String> tags = new HashSet<>();
```

Comment:
you should not put this new attribute in CommonProperties as it would be assigned to both Job and Task.

Instead create a workflowTags attribute in the Job class, with associated getter and setter